flow/designs/gf12/bp_single/config_hier.mk

```diff
@@ -3,22 +3,20 @@ export DESIGN_NAME = bsg_chip
 export PLATFORM    = gf12
 
 export SYNTH_HIERARCHICAL = 1
-FLOW_VARIANT ?= hier
 export RTLMP_FLOW = True
-
 #
 # RTL_MP Settings
 export RTLMP_MAX_INST = 30000
 export RTLMP_MIN_INST = 10000
 export RTLMP_MAX_MACRO = 24 
 export RTLMP_MIN_MACRO = 4
-#
+export RTLMP_DEAD_SPACE = 0.10
```

Comment:
Where are these magic numbers coming from?  This isn't "no human in the loop"

Reply:
These set the min and max  for cluster size   wrt to number of instances and macros.  These are fairly decent defaults -- I agree that most of the config should just use the defaults  and just have a few sample designs with option settings to show the use case

Also with hier rtlmp the dead_space parameter is auto tuned as part of the running the macro placer.

Reply:
Filed Jira to track [OR-1593](https://openroad.atlassian.net/browse/OR-1593 )  to track this